chapters/sampling-randomization-power.tex

```diff
@@ -398,8 +450,77 @@ \subsection{Minimum sample size}
 than an output for reporting requirements.
 At the end of the day, you will probably have reduced
 the complexity of your experiment significantly.
```

Comment:
reduced complexity by doing what?

Reply:
getting rid of all the silly stuff your PI wanted to do before you showed them power calculations ;-)